packages/genesys-spark-components/src/components/beta/gux-avatar/tests/gux-avatar.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { newSpecPage } from '@test/specTestUtils';
+import { GuxAvatar } from '../gux-avatar';
+
+describe('gux-avatar', () => {
+  describe('#render', () => {
+    [
+      `<gux-avatar-beta></gux-avatar-beta>`,
+      `<gux-avatar-beta name="John Doe"></gux-avatar-beta>`,
+      `<gux-avatar-beta><img slot="image" src="https://i.pravatar.cc/300" /></gux-avatar-beta>`,
+      `<gux-avatar-beta name="John Doe"><img slot="image" src="https://i.pravatar.cc/300" /></gux-avatar-beta>`
```

Comment:
I think the tests for the invalid versions should be checking that a warning log was made not that they rendered.